Kerdy/Kerdy/Event/View/EventTableViewCell.swift

```diff
@@ -49,36 +68,63 @@ final class EventTableViewCell: UITableViewCell {
     }
 
     private func setLayout() {
-        contentView.addSubview(tagStackView)
-        contentView.addSubview(titleLabel)
-        contentView.addSubview(progressLabel)
         contentView.addSubview(eventImage)
+        contentView.addSubview(titleLabel)
+        contentView.addSubview(dDayLabel)
+        contentView.addSubview(priceLabel)
+        contentView.addSubview(divideLine)
+        contentView.addSubview(placeLabel)
+        contentView.addSubview(tagStackView)
 
-        tagStackView.snp.makeConstraints {
+        eventImage.snp.makeConstraints {
+            $0.height.equalTo(180)
+            $0.width.equalTo(326).priority(250)
+            $0.top.equalToSuperview().offset(10)
+            $0.leading.equalToSuperview().offset(17)
+            $0.trailing.equalToSuperview().offset(-17)
```

Comment:
leading, trailing이 같은 경우 아래처럼 사용하는건 어떨까욤?
```suggestion
            $0.horizontalEdges.equalToSuperview().inset(17)
```